### DISCUSSION TITLE
Behaviour of "When opening the app, show … Last viewed note"

### DISCUSSION THREAD

**Discussion by retoter** [standardnotes/forum#2891] [General]
On macOS 13.0 the latest StandardNotes always opens the top note from the list, despite the chosen setting "Last viewed note" in "When opening the app, show …".

On Android (latest CalyxOS) however, the opened app shows me only just the list of notes.

Since I mainly use one principal note (95 %) and the other notes very rarely, it would be very handy to always open the same note – even after a restart/crash etc.

**Comment by effieeee** [comment]
Can reproduce for iOS and Android :(
